waku/v2/peerstore/waku_peer_store.go

```diff
@@ -155,6 +155,11 @@ func (ps *WakuPeerstoreImpl) AddPubSubTopic(p peer.ID, topic string) error {
 	if err != nil {
 		return err
 	}
+	for _, t := range existingTopics {
+		if t == topic {
+			return errors.New("Pubsub topic already exists")
```

Comment:
Now that i am taking a relook, i am wondering if it makes sense to silently return rather than an error. 
That way no harm is done

Reply:
Ok, will change this.